src/plugins/autoColumnSize/autoColumnSize.js

```diff
@@ -44,27 +45,31 @@ const COLUMN_SIZE_MAP_NAME = 'autoColumnSize';
  * autoColumnSize: {syncLimit: '40%'},
  * ```
  *
- * The plugin uses {@link GhostTable} and {@link SamplesGenerator} for calculations.
- * First, {@link SamplesGenerator} prepares samples of data with its coordinates.
- * Next {@link GhostTable} uses coordinates to get cells' renderers and append all to the DOM through DocumentFragment.
+ * The plugin uses {@link ghost-table GhostTable} and {@link samples-generator SamplesGenerator} for calculations.
+ * First, {@link samples-generator SamplesGenerator} prepares samples of data with its coordinates.
+ * Next {@link ghost-table GhostTable} uses coordinates to get cells' renderers and append all to the DOM through DocumentFragment.
```

Comment:
It was a workaround in the early development stage and was on the stick of jsdoc and vuepress integration. Currently, it may use `@/api/GhostTable.md` syntax, which I love.
As long as it works, it does not need to be changed with high priority. Reassuming, I would like to create an issue for that and plan to do it in the next iterations.

Reply:
`@/api/GhostTable.md` is ok, when we link to API from guides. API refs should be autolinked, we know the context.

So we introduce lots of unnecessary changes to our JSDoc just to change it later?

Reply:
Let's do it before the initial release @Siemienik , thank you.

Reply:
Done in https://github.com/handsontable/handsontable/pull/8496